_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,524 @@
+                <input class="field-input input-text input-alt has-error" type="text" id="full_name_simple_err" name="full_name_simple_err" placeholder="e.g. Jane Doe" value="JD" required>
+                <div class="field-message has-error">
+                    <span class="field-message-content">
+                        You must provide your real name.
+                    </span>
+                </div>
+            </div>
+            <div class="field">
+                <label class="field-label" for="full_name_simple_suc">Your name (simple text field, success)</label>
+                <input class="field-input input-text input-alt has-success" type="text" id="full_name_simple_suc" name="full_name_simple_suc" placeholder="e.g. Jane Doe" value="Jane Doe" required>
+            </div>
+            <div class="field">
+                <label class="field-label" for="full_name_simple_disabled">Your name (disabled simple text field)</label>
+                <input class="field-input input-text input-alt" type="text" id="full_name_simple_disabled" name="full_name_simple_disabled" placeholder="e.g. Jane Doe" required disabled>
+            </div>
+        </fieldset>
+    </form>
+</div>
+
+<h4 class="hd-6 example-set-hd">Search</h4>
+<div class="example-set">
+    <form class="form">
+        <fieldset class="form-group">
+            <legend class="form-group-hd sr-only">Search examples</legend>
+            <div class="field">
+                <label class="field-label" for="search_01">Search (with visible label)</label>
+                <input class="field-input input-text" type="search" id="search_01" name="search_01" placeholder="Search this website">
+                <button class="btn btn-primary btn-small" type="button">Search</button>
+            </div>
+            <div class="field">
+                <label class="field-label sr-only" for="search_02">Search</label>
+                <input class="field-input input-text" type="search" id="search_02" name="search_02" placeholder="Search this website" aria-describedby="search_02_hint">
+                <button class="btn btn-primary btn-small" type="button">Search</button>
+                <div class="field-hint" id="search_02_hint">
+                    <p>(This field has a label that's hidden yet still accessible.)</p>
+                </div>
+            </div>
+            <div class="field">
+                <label class="field-label" for="search_03">Search (with visible label and icon)</label>
+                <input class="field-input input-text" type="search" id="search_03" name="search_03" placeholder="Search this website">
+                <button class="btn btn-primary btn-small" type="button">
+                    <svg class="icon icon-search" title="Search"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/public/images/edx-svg/svgdefs.svg#icon-search"></use></svg>
```

Comment:
@clrux think what you want here is  a ```<title>``` element not a title attriubute.  To improve support across browsers and AT, you should also add a ```role=image``` to the svg element and an ```aria-labelledby``` that references the ```<title>``` element.